datasets/kwaliteitsmonitor/kwaliteitsmonitorWonenStandplaats/v1.0.0.json

```diff
@@ -0,0 +1,169 @@
+{
+  "id": "kwaliteitsmonitorWonenStandplaats",
+  "type": "table",
+  "title": "Kwaliteitsmonitor Wonen Standplaats",
+  "version": "1.0.0",
+  "description": "Per standplaats de kenmerken van de woningvoorraad van de gemeente Amsterdam.",
+  "auth": "FP/WONEN",
+  "reasonsNonPublic": [
+    "5.1 2f: Zwaarwegend belang: vertrouwelijke of concurrentiegevoelige bedrijfs- en fabricagegegevens"
+  ],
+  "status": "beschikbaar",
+  "creator": "Datateam Stedelijke Ontwikkeling en Beheer",
+  "publisher": "Datateam Stedelijke Ontwikkeling en Beheer",
+  "authorizationGrantor": "datateamstedelijkeonwikkelingenbeheer@amsterdam.nl",
+  "theme": [
+    "Wonen"
+  ],
+  "owner": "Gemeente Amsterdam",
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "required": [
+      "schema",
+      "id"
+    ],
+    "mainGeometry": "spsGeomMiddel",
+    "display": "id",
+    "properties": {
+      "id": {
+        "type": "integer",
+        "description": "Unieke id van de standplaats"
+      },
+      "schema": {
+        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+      },
+      "adres": {
+        "type": "string",
+        "description": "Adres van de standplaats"
+      },
+      "huisnummer": {
+        "type": "integer",
+        "description": "Huisnummer van de standplaats"
+      },
+      "huisletter": {
+        "type": "string",
+        "description": "Huisletter van de standplaats"
+      },
+      "huisnummertoevoeging": {
+        "type": "string",
+        "description": "Huisnummertoevoeging van de standplaats"
+      },
+      "postcode": {
+        "type": "string",
+        "description": "Postcode van de standplaats"
+      },
+      "woonplaats": {
+        "type": "string",
+        "description": "Woonplaats van de standplaats"
+      },
+      "bagNagId": {
+        "type": "string",
+        "description": "Unieke nummeraanduiding van de standplaats"
+      },
+      "bagNagVolgnummer": {
+        "type": "integer",
+        "description": "Volgnummer van de nummeraanduiding behorende bij de standplaats"
+      },
+      "spsIdentificatie": {
+        "type": "string",
+        "description": "Unieke aanduiding van de standplaats"
+      },
+      "bagTypeadresseerbaarobject": {
+        "type": "string",
+        "description": "-"
+      },
+      "bagStatus": {
+        "type": "string",
+        "description": "De fase van de levenscyclus van een standplaats waarin de betreffende standplaats zich bevindt"
+      },
+      "bagStatusCode": {
+        "type": "string",
+        "description": "-"
+      },
+      "bsdLigtInBeschermdGebied": {
+        "type": "string",
+        "description": "Ja/Nee indicator welke aangeeft of de standplaats in een beschermd stads- of dorpgezicht valt"
```

Comment:
Ja/nee kan beter een `boolean` worden. Dat is compacter, sneller en makkelijker te bevragen.

Reply:
Het veld is geen boolean in overleg met opdrachtgever.